src/fXmlRpc/Parser/NativeParser.php

```diff
@@ -31,16 +30,28 @@
 
 final class NativeParser implements ParserInterface
 {
-    public function __construct()
+    /**
+     * @var bool
+     */
+    private $validateResponse;
+
+    public function __construct($validateResponse = true)
     {
         if (!extension_loaded('xmlrpc')) {
             throw MissingExtensionException::extensionMissing('xmlrpc');
         }
+        $this->validateResponse = $validateResponse;
     }
 
     /** {@inheritdoc} */
     public function parse($xmlString, &$isFault)
     {
+        libxml_use_internal_errors(true);
```

Comment:
I think about move this to XmlChecker, but I didn't do this because this method is used in XmlReaderParser and I want avoid redundant call. Additionally I think this is not responsibility of the XmlChecker to setting error level.